server/integration/testsuite/pom.xml

```diff
@@ -23,26 +23,24 @@
             <artifactId>arquillian-junit-container</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.wildfly.arquillian</groupId>
-            <artifactId>wildfly-arquillian-container-managed</artifactId>
+            <groupId>org.infinispan.arquillian.container</groupId>
+            <artifactId>infinispan-arquillian-impl</artifactId>
             <exclusions>
```

Comment:
I'm adding it back in a separate profile below. They can't be in "default" profile because they would conflict with domain mode dependencies.